Description: Makefile.am: install-data-hook depends on install-exec
 Calling $(DESTDIR)$(bindir)/vnstat$(EXEEXT) fails when
 installation of this program is not yet finished.
Author: Adrian Bunk <bunk@debian.org>

--- vnstat-1.15.orig/Makefile.am
+++ vnstat-1.15/Makefile.am
@@ -93,7 +93,7 @@ release: distcheck
 	rm -fr $(PACKAGE_NAME)-$(PACKAGE_VERSION)
 	ls -l $(PACKAGE_NAME)-$(PACKAGE_VERSION).tar.gz
 
-install-data-hook:
+install-data-hook: install-exec
 	$(DESTDIR)$(bindir)/vnstat$(EXEEXT) --showconfig >vnstat.conf && \
 	  $(MKDIR_P) $(DESTDIR)$(sysconfdir) && \
 	  mv -f vnstat.conf $(DESTDIR)$(sysconfdir)/vnstat.conf
